sites/packworld.com/server/routes/website-section.js

```diff
@@ -11,6 +11,7 @@ const superCategory = require('@pmmi-media-group/package-global/templates/websit
 
 const { newsletterState } = require('@pmmi-media-group/package-global/middleware/newsletter-state');
 
+const description = require('@parameter1/base-cms-marko-web/utils/published-content/description');
```

Comment:
This is likely what is actually causing your issues, should be able to remove this and all the es-lint disable comments and will **potentially** pass all the integration tests.